### DISCUSSION TITLE
Reuse database

### DISCUSSION THREAD

**Discussion by cbirkenbeul** [teslamate-org/teslamate#2639] [General]
Hi,
i installed the postgres database on one host and the application is running in my Kubernetes cluster. I rebuild the cluster and redeploy teslamate. unfortunately teslamate is not able to connect to the database anymore, because It cannot create the database. Is there a way to work around this? I mean besides of backup, delete and restore the current db.

Regards

**Comment by balajeek** [comment]
i have the same situation, i already have postgres db, and i want to re use it, its not in docker, but teslamate is in docker.
